spec/std/file_utils_spec.cr

```diff
@@ -157,6 +157,18 @@ describe "FileUtils" do
       end
     end
 
+    it "deletes read-only file" do
+      with_tempfile("rm_r") do |path|
+        Dir.mkdir(path)
+        Dir.mkdir(File.join(path, ".git"))
```

Comment:
Is there a particular reason to call this dir `.git`? It's just a bit confusing to see a dot name for no apparent reason. The read-only mode is because of `chmod`, not the name, right?
More generic `a/b/c` path names are IMO preferable for such tests.

Reply:
That's the real-world scenario where you'd find these ¯\\\_(ツ)_/¯
Removed.